### ISSUE TITLE
Allow passing ranges for `--index-memory-cells`, e.g. `1,3-5,9,10-13`

### ISSUE THREAD

**Issue by reeelix** [LMH01/alpha_tui#66]
Allow passing ranges for `--index-memory-cells`.  
For example `--index-memory-cells 1,3-5,9,10-13` should be treated like `--index-memory-cells 1,3,4,5,9,10,11,12,13`. This makes it more convenient to pass a big amount of memory cells.  
  
This change might require some refactoring first, as currently the commandline parameter ``--index-memory-cells` is parsed into a vector of numbers directly by Clap.

**Comment by reeelix** [comment]
not urgent

**Comment by LMH01** [comment]
Yeah that is another good idea, but implementing it is not so easy, I will have to see how it can be done.

**Comment by LMH01** [comment]
I took a look at how this could be implemented but unfortunately it seems not easily possible using clap build in features. The approach that is currently commited on the branch does not work.
I might look for a workaround in the future. You can provide clap with a value parser but that is only used for a single value and not for the whole thing.
